mediaqueries-5/Overview.bs

```diff
@@ -3216,6 +3217,306 @@ Automatic handling of User Preferences</h3>
 		allows for unlimited data or is on a metered plan.
 	</div>
 
+<h2 id=auto-pref>
+Script Control of User Preferences</h3>
+
+It is common for website authors to want to respect the user's system preferences while also allowing
```

Comment:
Probably needs a bit of a better introduction here. Perhaps with more from https://wicg.github.io/web-preferences-api/#sec-intro